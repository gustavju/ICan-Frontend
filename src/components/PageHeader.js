import React from 'react';

const PageHeader = ({ title, subTitle, data }) => (
    <div className="page-header">
        <div className="content-container">
            <h2 className="page-header__title">{title}</h2>
            <h3 className="page-header__subtitle"><span>{data}</span> {subTitle}</h3>
        </div>
    </div>
);

export default PageHeader;
