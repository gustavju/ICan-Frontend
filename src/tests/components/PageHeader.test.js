import React from 'react';
import { shallow } from 'enzyme';
import PageHeader from '../../components/PageHeader';

test('should render the PageHeader', () => {
    const wrapper = shallow(<PageHeader title="title" subTitle="subTitle" data="1" />);
    expect(wrapper).toMatchSnapshot();
});