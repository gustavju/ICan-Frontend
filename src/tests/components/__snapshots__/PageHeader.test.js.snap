// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the PageHeader 1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h2
      className="page-header__title"
    >
      title
    </h2>
    <h3
      className="page-header__subtitle"
    >
      <span>
        1
      </span>
       
      subTitle
    </h3>
  </div>
</div>
`;
